modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/CheckboxTag.java

```diff
@@ -41,10 +41,26 @@ public boolean getChecked() {
 		return _checked;
 	}
 
+	public String getCssClass() {
```

Comment:
You  don't need to add `getCssClass` nor `seCssClass` since they're already defined in `BaseContainer` and this class extends from it.